src/app/zap-templates/zcl/data-model/chip/color-control-cluster.xml

```diff
@@ -107,6 +100,24 @@ limitations under the License.
 
     <globalAttribute side="either" code="0xFFFD" value="6"/>
 
+    <features>
+      <feature bit="0" code="HS" name="Hue/Saturation" summary="Supports color specification via hue/saturation.">
```

Comment:
HueAndSaturation became Hue/Saturation

Reply:
That's wrong and needs to be fixed.  This is long since shipped as API and cannot change.